ext/msgpack/packer.c

```diff
@@ -37,6 +38,8 @@ void msgpack_packer_static_init()
 #endif
 
     s_call = rb_intern("call");
+    ePackError = rb_define_class_under(mMessagePack, "PackError", rb_eStandardError);
+    rb_define_attr(ePackError, "receiver", 1, 0);
```

Comment:
This should be named something better, just using `receiver` for now to match current internal usage.